app/assets/data/bebbo/sponsers.ts

```diff
@@ -0,0 +1,361 @@
+
+import { nmSponserLogo } from "@dynamicImportsClass/dynamicImports";
+
+const sponsersall = [
+  {
+    id: 6,
+    type: 'Country',
+    title: "Albania",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      name: "IPH.png",
+      alt: "IPH"
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Albania-Albanian"
+    ]
+  },
+  {
+    id: 106,
+    type: "Country",
+    title: "Belarus",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      name: "Минздрав.png",
+      alt: "Ministry of Health Belarus "
+    },
+    country_national_partner: {
+      name: "Минобр.png",
+      alt: "Ministry of Education Belarus "
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Belarus-Belarusian",
+      "Belarus-Russian"
+    ]
+  },
+  {
+    id: 11,
+    type: "Country",
+    title: "Bulgaria",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Bulgaria-Bulgarian"
+    ]
+  },
+  {
+    id: 16,
+    type: "Country",
+    title: "Greece",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Greece-Arabic",
+      "Greece-Greek",
+      "Greece-Persian"
+    ]
+  },
+  {
+    id: 26,
+    type: "Country",
+    title: "Kyrgyzstan",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      name: "Logo MoES KG White Side.png",
+      alt: "MoES Kyrgyzstan "
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Kyrgyzstan-Kyrgyz",
+      "Kyrgyzstan-Russian"
+    ]
+  },
+  {
+    id: 146,
+    type: "Country",
+    title: "Moldova",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    make_available_for_mobile: 0,
+    language: [
+      "Moldova-Romanian"
+    ]
+  },
+  {
+    id: 31,
+    type: "Country",
+    title: "Montenegro",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      name: "EU logo za aplikaciju.png",
+      alt: "EU"

```

Comment:
Let's add a descriptive and meaningful text to improve accessibility.

Reply:
Can we improve API then? When users with disabilities access this code it will be hard for them to understand the logic.